dapp-example/README.md

```diff
@@ -0,0 +1,11 @@
+# Dapp example
+
+
```

Comment:
nit: I feel we should expand the README a bit more.
Though I don't want to block on this so do what you can and let's capture the rest in a ticket

- [ ] Note the location of contract code. Since its the standard Greet contract you should be able to add a link
- [ ] Add some screen shots of the flow so users know what to expect
- [ ] Please call out the scenarios supported. Similar to that in the PR description e.g. HBAR transfer etc